App.tsx

```diff
@@ -1,4 +1,3 @@
-import "@/background/BackgroundTasks";
 import { Notification } from "@notifee/react-native";
```

Comment:
non, il faut laisser la 1ère ligne pour que le background soit géré correctement

```suggestion
import "@/background/BackgroundTasks";
import { Notification } from "@notifee/react-native";
```